crates/turborepo-lib/src/commands/daemon.rs

```diff
@@ -59,6 +62,43 @@ pub async fn main(command: &DaemonCommand, base: &CommandBase) -> anyhow::Result
     Ok(())
 }
 
+pub async fn daemon_server(
+    base: &CommandBase,
+    idle_time: &Option<String>,
+) -> Result<(), DaemonError> {
+    let log_file = {
+        let directories = directories::ProjectDirs::from("com", "turborepo", "turborepo")
+            .expect("user has a home dir");
+
+        let folder =
+            turborepo_paths::AbsoluteNormalizedPath::new(directories.data_dir()).expect("absolute");
+
+        let hash = format!("{}-turbo.log", base.repo_hash());
+
+        let logs = turborepo_paths::ForwardRelativePath::new("logs").expect("forward relative");
+        let file = turborepo_paths::ForwardRelativePath::new(&hash).expect("forward relative");
+
+        folder.join(logs).join(file)
+    };
+
+    let repo_root =
+        turborepo_paths::AbsoluteNormalizedPathBuf::new(base.repo_root.clone()).expect("absolute");
+
+    let timeout = idle_time
+        .as_ref()
+        .map(|s| {
+            go_parse_duration::parse_duration(s).map_err(|_| DaemonError::InvalidTimeout(s.clone()))
+        })
+        .transpose()?
+        .map(|d| Duration::from_nanos(d as u64))
+        .unwrap_or_else(|| Duration::from_secs(60 * 60 * 4));
```

Comment:
This duration should probably be extracted as a constant since `Duration::from_secs` is now a const function.

Reply:
Opted instead to use the clap default_value_t flag to we can document this in the cli.